cassandra/src/test/groovy/io/micronaut/configuration/cassandra/CassandraConfigurationSpec.groovy

```diff
@@ -35,86 +39,78 @@ class CassandraConfigurationSpec extends Specification {
 
         expect: "No beans are created"
         !applicationContext.containsBean(CassandraConfiguration)
-        !applicationContext.containsBean(Cluster)
+        !applicationContext.containsBean(CqlSession)
 
         cleanup:
         applicationContext.close()
     }
 
     void "test single cluster connection"() {
         given:
-        // tag::single[]
+        EmbeddedCassandraServerHelper.startEmbeddedCassandra();
         ApplicationContext applicationContext = new DefaultApplicationContext("test")
         applicationContext.environment.addPropertySource(MapPropertySource.of(
                 'test',
-                ['cassandra.default.clusterName': "ociCluster",
-                 'cassandra.default.contactPoint': "localhost",
-                 'cassandra.default.port': 9042,
-                 'cassandra.default.maxSchemaAgreementWaitSeconds': 20,
-                 'cassandra.default.ssl': true]
+                ['cassandra.datasource.default.basic.contact-points'                        : ["localhost:9142"],
```

Comment:
So with this upgrade, old configs would break entirely? Is that accurate? One thing I was thinking about was adding some sort of mapping functions to graft the old config onto the new values: do you think that's doable?